core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -453,7 +453,31 @@ contract PricelessPositionManager is FeePayer {
         oracle.requestPrice(priceIdentifer, requestedTime);
     }
 
-    function _getOraclePrice(uint requestedTime) internal view returns (FixedPoint.Unsigned memory) {
+    function _payFinalFee() internal {
+        // Send the fee payment.
+        FixedPoint.Unsigned memory totalPaid = super.payFinalFees();
```

Comment:
Why are you using `super` here? Why not just call `payFinalFees()`? I'm assuming you're not trying to pass the call up the inheritance tree -- you just want to call the most most derived/virtual `payFinalFees()` function, right?

Reply:
ah this was just a typo in copying the code initially from `PricelessPositionManager.payFees()`